vsts/fortify/filter.txt

```diff
@@ -189,4 +189,5 @@ E10238AE08E9048E3D0C0590CA43AC01
 EB88D40BFA36002BFBBC2D26501D5111
 02BEB7B8FC297FE5C8110708272DA2B4
 11F880EB6AEABBFF8CC1B0BC6A3FB21D
-C5C6C2943C1AC9DF47725F75B2163020
\ No newline at end of file
+C5C6C2943C1AC9DF47725F75B2163020
+D9A3DBA80AB3999F4F8C1F1AA9E44B7B
```

Comment:
For the uninitiated, we use Fortify to scan for security vulnerabilities and other code smells. This file contains the false positives that Fortify complains about, but have manually been verified to be fine.

Reply:
Is there a way to leave a comment in this file to that effect?

Reply:
Yes, and the comment is in the file above all these random strings